_summaries/goodfellow2013maxout_1.md

```diff
@@ -1,19 +1,92 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: goodfellow2013maxout
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: Zayne-sprague
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+*TODO - how is this different than max-pooling for CNNs?  Adding this would be helpful to the review.
+*TODO - add the visuals where marked
+*TODO - add equations where marked
+*TODO - break up sections a bit neater.
+
+
+**_Main Idea_**
+
+Taking the element-wise max of an input across multiple weight matrices produces a non-linear activation function
+that can learn any convex function and take advantage of dropout layers while improving accuracy and allowing deeper networks.
+
+**_How is it realized_**
+
+Maxout networks use "Maxout Layers".  
+
+Maxout Layers have multiple weight matrices and biases for the same input!  
+(tip: Think of convolutional feature maps, in MLPs or linear layers they are just extra weight matrices and biases)
+
+Maxout Layers takes the values from all the weight matrices in its layer, does the element wise multiplication with the
+input per weight matrix, and then does an element wise maximum across all the different activations for that input.
```

Comment:
I think it would be help to include the equation that defines the maxout function.   In particular, we could include (perhaps a paraphrased version of) the equation at the top of page 2 column 2.  The current description makes sense to me, but would be confusing if I had not read the paper beforehand.

Reply:
Very good call, adding them now.